src/test/java/org/folio/edgecommonspring/util/ApiKeyHelperImplTest.java

```diff
@@ -57,7 +57,7 @@ void testParamOnly() {
       .setField(apiKeyHelperImpl, "apiKeySources", "PARAM");
     apiKeyHelperImpl.init();
     MockHttpServletRequest request = new MockHttpServletRequest();
-    request.setParameter("apikey", "testKeyParam");
```

Comment:
probably worth add one more test when apiKey in camel case will be used

Reply:
Added